src/proto/project.proto

```diff
@@ -0,0 +1,85 @@
+syntax = 'proto3';
+package gooseai;
+option go_package = "./;project";
+
+enum ProjectAccess {
+  PROJECT_ACCESS_PRIVATE = 0; // Private access, only owner organization can access
+  PROJECT_ACCESS_PUBLIC = 1;  // Public access, anyone can access
+}
+
+enum ProjectStatus {
+  PROJECT_STATUS_INACTIVE = 0; // Inactive project
+  PROJECT_STATUS_ACTIVE = 1;   // Active project
+}
+
+enum ProjectAssetUse {
+  PROJECT_ASSET_USE_INPUT = 0;        // Asset is used as an input for the project
+  PROJECT_ASSET_USE_OUTPUT = 1;       // Asset is an output from the project
+  PROJECT_ASSET_USE_INTERMEDIATE = 2; // Asset is an output from an intermediate step of the project
+  PROJECT_ASSET_USE_PROJECT = 3;      // Asset is used as the project file for the project
+}
+
+message ProjectAsset {
+  uint64 id = 1;           // ID of the asset
+  string URI = 2;          // The URI to the asset
+  ProjectAssetUse use = 3; // The use of the asset with respect to the project
+}
+
+message Project {
+  uint64 id = 1;                     // ID of the project
```

Comment:
suggest an uuid v4 and translating that to a binary lookup on the db/backend side.

Reply:
If we use a UUID here, then we should be using across the board for these in my opinion no reason to translate UUID to ID.